cache/class-vary-cache.php

```diff
@@ -114,9 +114,8 @@ public static function set_group_for_user( $group, $value ) {
 	 */
 	public static function is_user_in_group( $group ) {
 		self::parse_group_cookie();
-
 		// The group isn't defined, or the user isn't in it.
-		if ( ! array_key_exists( $group, self::$groups ) ) {
+		if ( ! array_key_exists( $group, self::$groups ) || null == self::$groups[ $group ] ) {
```

Comment:
Should probably use a strict check here:

```suggestion
		if ( ! array_key_exists( $group, self::$groups ) || null === self::$groups[ $group ] ) {
```

Reply:
You're too fast! I changed it to a trim() to catch the empty string as well.